api/v1alpha/src/admin/index.js

```diff
@@ -188,4 +188,13 @@ admin.post('/projects/:projectId/admin::custom', async (req, res) => {
     }
 });
 
+admin.get('/projects/:projectId/admin/isAdmin', async(req, res) => {
```

Comment:
I would suggest changing `/admin/isAdmin` to something that closely represents the action of the client and fits a custom HTTP verb logic like e.g POST `/auth::isAdmin`. Also should this map to the name of the admin group [Data Producers](https://github.com/GoogleCloudPlatform/datashare-toolkit/tree/master/api#data-producers)

Reply:
Change to post and making /auth::custom - handle case as 'isDataProducer'.